kbs/src/http/attest.rs

```diff
@@ -16,7 +16,7 @@ use serde_json::json;
 
 static KBS_MAJOR_VERSION: u64 = 0;
 static KBS_MINOR_VERSION: u64 = 1;
-static KBS_PATCH_VERSION: u64 = 0;
+static KBS_PATCH_VERSION: u64 = 1;
```

Comment:
Do you think we should make KBS forward compatible? Another view is still kbs is not stable release like `1.x`, thus we could live that?

cc @fitzthum @mkulke

Reply:
we have not talked about this too much and I think there's two dimensions: version compatibility with `trustee` and `guest-components` releases and version compatibility with the KBS protocol versions. Currently, the `version` check ( I did not change that in https://github.com/confidential-containers/trustee/commit/c42dea0367b2767d8d2a220e4d4a42e901626846) is so that RCAR clients cannot use a newer protocol version but in real life only an exact version match works today (RCAR client 1.0.0 wont work with KBS 1.0.1).

Reply:
IMO it would be an improvement to have inflexible, but predictable protocol versioning for the time being. Supporting more options than a 1:1 version match might be impeding the quick iterations on the protocol before a 1.0 release of kbs

Reply:
Makes sense. However, since we now have `trustee` and `guest-components` releases decoupled we need to document the version dependencies in the release notes when the protocol version changes.

Reply:
> Supporting more options than a 1:1 version match might be impeding the quick iterations on the protocol before a 1.0 release of kbs

I think we are currently evidencing this :-) I could add change the version check to be an exact match and we go ahead with the current `tee-evidence` as `Value`?

Reply:
I pushed a separate commit for requiring an exact version match.